A32NX/html_ui/Pages/A32NX_Utils/NXFlightGuidance.js

```diff
@@ -0,0 +1,126 @@
+// Lives here as a Util for now so it can be cleanly separate from FMCMainDisplay.
```

Comment:
We should drop this again, until LNAV is ready.

Reply:
Agree